CONTRIBUTING.rst

```diff
@@ -32,7 +32,7 @@ To get started:
 2. Ensure you have Docker running on your local machine.
 3. Ensure you have VSCode running on your local machine.
 4. In Visual Studio Code,
-   click on the quick actions Status Bar item in the lower left corner.
+    click on the quick actions Status Bar item in the lower left corner.
 5. Then select "Remote Containers: Open Repository In Container".
```

Comment:
Seems Microsoft has changed `"Remote Containers: Open Repository In Container"` to `"Remote Containers: Clone Repository In Container Volume"`

Reply:
Great catch, thanks @samukweku!